hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/snapshot/TestOMSnapshotDeleteResponse.java

```diff
@@ -56,7 +55,7 @@ public class TestOMSnapshotDeleteResponse {
   @TempDir
   private Path folder;
   
-  private OMMetadataManager omMetadataManager;
+  private OmMetadataManagerImpl omMetadataManager;
```

Comment:
We can retain `omMetadataManager` object to be of type `OMMetadataManager` as is.